src/main/java/raisetech/StudentManagement/Student.java

```diff
@@ -1,23 +1,18 @@
 package raisetech.StudentManagement;
 
-public class Student {
-
-  private  String name;
-  private int age;
+import lombok.Getter;
+import lombok.Setter;
 
-  public String getName() {
-    return name;
-  }
+@Getter
+@Setter
 
-  public void setName(String name) {
-    this.name = name;
-  }
-
-  public int getAge() {
-    return age;
-  }
+public class Student {
 
-  public void setAge(int age) {
-    this.age = age;
-  }
+  private String id;
+  private String name;
+  private String nickname;
+  private int age;
+  private String gender;
+  private String mailaddress;
+  private  String region;
```

Comment:
フォーマットしましょう
また、キャメルケースに揃えましょう。
```suggestion
  private String mailAddress;
  private String region;
```